packages/myst-execute/tests/execute.yml

```diff
@@ -0,0 +1,200 @@
+title: executionTransform
```

Comment:
Can we also add an ipynb test case? Maybe that is more of an end-to-end test later (e.g. if it requires the tree to be parsed)?